100=request.execute()
101=URLRequiredException(): HTTP request: {} with URL: {} cannot be opened!!!
102=ConnectTimeoutException(): Despite a delay of {} seconds, the HTTP request: {} with URL: {} generated a connection timeout exception!!!
103=ReadTimeoutException(): Despite delaying {} seconds, HTTP request: {} with URL: {} generated a read timeout exception!!!
104=ConnectionException(): Unable to connect with HTTP request: {} on URL: {}!!!
105=HTTPException(): HTTP request: {} with URL: {} returned HTTP status code: {} with reason: {}!!!
106=TooManyRedirectsException(): HTTP request: {} with URL: {} contains too many redirects!!!
107=RequestException(): HTTP request: {} with URL: {} threw a general exception!!! : {}

120=request.download()
121=Download file: {}, elapsed time: {}:{}:{}.{}.
122=The HTTP request: {} while downloading generated the following error: {}
123=The HTTP request: {} while downloading on URL: {} returned HTTP status code: {} with content: {}!!!
124=The HTTP request: {} while downloading on URL: {} threw a general exception with content: {}!!!

130=request.upload()
131=Upload file: {} in {} requests, elapsed time: {}:{}:{}.{}.
132=The HTTP request: {} while uploading generated the following error: HTTP status code: {} - Response content: {}
133=The HTTP request: {} while uploading generated the following error: {}
134=The HTTP request: {} while uploading on URL: {} returned HTTP status code: {} with content: {}!!!
135=The HTTP request: {} while uploading on URL: {} threw a general exception with content: {}!!!

140=OptionsHandler.callHandlerMethod()
141=An unexpected exception: {} has occurred, producing a series of exceptions: {}

150=OptionsManager.__init__()
151=OptionManger has been correctly initialized.

160=OptionsManager.loadSetting()
161=OptionManger has been correctly reloaded.

170=OptionsManager.saveSetting()
171=OptionManger saved options {} and logging {}.

180=OptionsManager.connect()
181=OptionManger successfully executed the OAuth2 Wizard for user {} and url {}.
182=OptionManger failed in OAuth2 Wizard for user {} / url {} with exception {} and trace {}.

